src/test/java/uk/gov/hmcts/reform/sscs/personalisation/SyaAppealCreatedAndReceivedPersonalisationTest.java

```diff
@@ -454,4 +454,25 @@ public void givenASyaAppealWithNoHearingArrangements_setHearingArrangementsForTe
                 result.get(AppConstants.HEARING_ARRANGEMENT_DETAILS_LITERAL));
     }
 
+    @Test public void getOptionalFieldTest() {
+
+        final Name emptyName = Name.builder().build();
+        final Name firstName = Name.builder().firstName("FIRST").build();
+        final Name lastName = Name.builder().lastName("LAST").build();
+        final Name bothName = Name.builder().firstName("FIRST").lastName("LAST").build();
+        final Name allName = Name.builder().title("MX").firstName("FIRST").lastName("LAST").build();
+
+        assertEquals("expected", syaAppealCreatedAndReceivedPersonalisation.getOptionalField(null, "expected"));
+        assertEquals("expected", syaAppealCreatedAndReceivedPersonalisation.getOptionalField(emptyName.getFullName(), "expected"));
+        assertEquals("expected", syaAppealCreatedAndReceivedPersonalisation.getOptionalField(emptyName.getFullNameNoTitle(), "expected"));
+        assertEquals("null FIRST null", syaAppealCreatedAndReceivedPersonalisation.getOptionalField(firstName.getFullName(), "expected"));
```

Comment:
There is zero guidance and no acceptance criteria on this and previous code explicitly tested that the name is rendered as "null null" if first and last names were null.
Personally, yes, I think if the name isn't entirely there we should use "Sir / Madam" so I'll check with... I don't know who - maybe Michael since he's content person?